daemon/graphdriver/driver.go

```diff
@@ -38,6 +38,7 @@ var (
 	priority = []string{
 		"aufs",
 		"btrfs",
+		"zfs",
```

Comment:
@LK4D4 Having it here is reasonable because:
1) devicemapper on a ZFS FS is a bad idea
2) ZFS is incompatible with overlay (most likely)
3) aufs still gets used on top of zfs if the kernel is aufs enabled